lib/Habitat.lua

```diff
@@ -11,9 +11,10 @@ local fs = import("./fs")
 local Habitat = {}
 Habitat.__index = Habitat
 
-function Habitat.new()
+function Habitat.new(flags)
```

Comment:
Can we make `Habitat` accept an optional `settings` object that has an optional `flags` field? There are already a number of things I want to expose for configuration and I don't want to shift around the API too much.